packages/backend/src/database/entities/catalog.ts

```diff
@@ -91,4 +91,11 @@ export type CatalogTagsTable = Knex.CompositeTableType<
     DbCatalogTagIn
 >;
 
+export type DbCatalogTagsMigrateIn = {
+    catalog_search_uuid: string;
+    tag_uuid: string;
+    created_by_user_uuid: string | null;
+    created_at: Date;
+};
+
```

Comment:
```suggestion
export type DbCatalogTagsMigrateIn = DbCatalogTag

```

Reply:
unless you want to Pick from the types to avoid passing more columns

Reply:
🤦